src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_img2img.py

```diff
@@ -195,6 +195,7 @@ def __call__(
         """
         if isinstance(prompt, str):
             batch_size = 1
+            prompt = [prompt]
```

Comment:
I'm planning on cleaning this up more generally when tackling https://github.com/huggingface/diffusers/issues/551